### PR TITLE
fixing #191, #190 #189

### PR DESCRIPTION
**Pull request by badr-r**
fixing issues #191 #190 #189

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -81,8 +82,8 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
     var totalProgress = 100;
     $scope.copyshow = false;
     //blockonomics_time_period is defined on JS file as global var
-    var totalTime = blockonomics_time_period * 60;
-    $scope.display_problems = true;
+    var totalTime = 600;
```
Comment (DarrenWestwood) [inline js/app.js:85]: In order to remove hard-coding the total time. Pass it with through with the order - `$scope.order.time.remaining` and `$scope.order.time.total`.

File: php/WC_Gateway_Blockonomics.php [hunk 2]
```diff
@@ -194,6 +194,7 @@ public function check_blockonomics_callback()
         }
         $address = isset($_REQUEST['get_order']) ? $_REQUEST['get_order'] : "";
         if ($address) {
+            $orders[$address]['remaining_time'] = $orders[$address]['timestamp'] + 600 - time();
```
Comment (DarrenWestwood) [inline php/WC_Gateway_Blockonomics.php:197]: The calculation for time remaining should still use the blockonomics timeperiod setting:
 `get_option('blockonomics_timeperiod', 10);`

**Comment by DarrenWestwood** [comment]
Please reference the issue numbers in the description to help link and track the issues.

**Comment by shivaenigma** [comment]
@badr-r Please write better more descriptive commit messages. Message like
>fixing error message
doesn't help. See other developer's commit messages for examples

**Comment by shivaenigma** [comment]
Lets make sure https://github.com/blockonomics/woocommerce-plugin/issues/190 is addressed properly. Better to test on few divi themes that we are having problems with [It should keep giving lite mode error]. You can enable/disable javascript via this addon https://addons.mozilla.org/en-US/firefox/addon/disable-javascript/

**Comment by shivaenigma** [comment]
I am closing this pull request as it has lots of conflicts. Master has changed a lot after merge. Will take these changes put seperately

**Pull request closed without merging**